src/jsTyping/types.ts

```diff
@@ -1,4 +1,3 @@
-/* @internal */

```

Comment:
This removal intentional?

Reply:
Yes. Otherwise we get test failures for our Public API tests after building tsserverlibrary.d.ts.

Reply:
They actually were public prior to the change to use project references:

https://github.com/Microsoft/TypeScript/blob/release-2.7/lib/tsserverlibrary.d.ts#L4845

Reply:
Huh. OK. I guess we should ask Ryan if he intended to make them private or not.

Reply:
I think the issue is that they weren't "public" in typescriptServices.d.ts/typescript.d.ts. Unfortunately, we can't make a file _conditionally_ internal.

@RyanCavanaugh - What's the best course of action here?

Reply:
So the jsTyping file now contributes to the services build? Huh. I imagine it should be public then, considering they were already public in the server library; unless we've got a good reason to keep it private (at which point it should be private in both).